frontends/web/src/app.jsx

```diff
@@ -156,12 +156,12 @@ export class App extends Component {
                         <Settings
                             path="/settings" />
                         {/* Use with TypeScript: {Route<{ deviceID: string }>({ path: '/manage-backups/:deviceID', component: ManageBackups })} */}
+                        {/* ManageBackups and DeviceSwitch need a key to trigger ComponentDidMount when devices change, to handle routing */}
```

Comment:
Strictly speaking, you use `componentWillMount ` in `ManageBackups`, not `componentDidMount`, thus `to trigger re-mounting` would be more correct.